src/chain/keysinterface.rs

```diff
@@ -163,15 +166,15 @@ impl KeysManager {
 					Err(_) => panic!("Your RNG is busted"),
 				};
 				let channel_master_key = master_key.ckd_priv(&secp_ctx, ChildNumber::from_hardened_idx(3)).expect("Your RNG is busted");
-				KeysManager {
+				(KeysManager {
 					secp_ctx,
 					node_secret,
 					destination_script,
 					shutdown_pubkey,
 					channel_master_key,
 
 					logger,
-				}
+				}, destination_secret.secret_key)
```

Comment:
Why is destination special here - the user needs to know cooperative close key as well. For now I think listing it in the comment is fine, but we can change the return value in the future.